tests/memdriver/mem_driver.go

```diff
@@ -0,0 +1,183 @@
+package memdriver
```

Comment:
We are building a `go-service-memory`, maybe we can use it for tests.

Reply:
that is great, use `go-service-memory` here is better.